docs/psoc6/quickref.rst

```diff
@@ -599,28 +558,74 @@ SPI object is created with default settings or settings of previous initializati
 ::
     
     from machine import SPI
-    spi = SPI(0, sck='P11_2', mosi='P11_0', miso='P11_1') # Default assignment: id=0, SCK=P11_2  ,MOSI=P11_0, MISO=P11_1
-    spi.init()
+    spi = SPI(baudrate=1000000, polarity=0,phase=0,bits=8,firstbit=SPI.MSB,sck='P6_2', mosi='P6_0', miso='P6_1')
```

Comment:
Add spaces between arguments for readabilty and consistency: 

```suggestion
    spi = SPI(baudrate=1000000, polarity=0, phase=0, bits=8, firstbit=SPI. MSB, sck='P6_2', mosi='P6_0', miso='P6_1')
```

Probably, the suggestion make the commit msg check fail. But first time I try this in Github.